www/src/components/shell/onboarding/cloud/InstallCli.js

```diff
@@ -1,36 +1,90 @@
-import { useContext } from 'react'
-import { P } from 'honorable'
-import { ArrowTopRightIcon, Button } from 'pluralsh-design-system'
+import { useContext, useState } from 'react'
+import { Div, Flex, P } from 'honorable'
+import { ArrowTopRightIcon, Button, Tab } from 'pluralsh-design-system'
 
 import CreateShellContext from '../../../../contexts/CreateShellContext'
 
+import CodeLine from '../../../utils/CodeLine'
+
 import OnboardingNavSection from '../OnboardingNavSection'
 
 import OnboardingCard from '../OnboardingCard'
 
 export default function InstallCli() {
   const { previous, next } = useContext(CreateShellContext)
+  const [tab, setTab] = useState(0)
 
   return (
     <>
-      <OnboardingCard title="Install Plural CLI">
-        <P
-          body1
-          color="text-light"
-          marginBottom="large"
-        >
-          Our <strong>Quickstart (CLI)</strong> documentation will bring you through the flow to get started on your local machine. After you're done, move on to the next step to experience the rest of what Plural has to offer.
+      <OnboardingCard title="Install CLI">
+        <Flex>
+          <Tab
+            active={tab === 0}
```

Comment:
if we number the tabs like this, we should at least define them as consts and make it more self-documenting

Reply:
Why not!